src/main/java/domain/Lotto.java

```diff
@@ -26,4 +47,24 @@ public String toString() {
         Collections.sort(numbers);
         return numbers.toString();
     }
+
+    @Override
+    public List<Integer> numbers() {
+        return Collections.unmodifiableList(numbers);
+    }
+
+    public int getMatchingNumberCount(List<Integer> comparingNumbers) {
+        int matchingNumberCount = 0;
+        for (Integer number : numbers) {
+            matchingNumberCount += booleanToInt(comparingNumbers.contains(number));
+        }
+        return matchingNumberCount;
+    }
```

Comment:
```suggestion
    public int getMatchingNumberCount(List<Integer> comparingNumbers) {
        return numbers.stream()
                .filter(comparingNumbers::contains)
                .toList()
                .size();
    }
```
현재 getMatchingNumberCount 메소드는 이와 같이 수정하면 아래의 booleanToInt 메소드를 이용하지 않을 수 있을 것 같아요

Reply:
stream 천재 무슨 일 . . . . .감사합니다 ㅠ_____ㅠ